content/en/docs/concepts/instrumenting.md

```diff
@@ -7,10 +7,10 @@ weight: 40
 ---
 
 In order to make a system observable, it must be **instrumented**: That is, the
```

Comment:
Thanks for the proposal. I'd prefer something like:

```suggestion
In order to make a system observable, it must be **instrumented**: That is, code from the
system's components must emit
```
Thoughts @cartermp et al.

If you agree, then drop the other changes.